src/ui/public/doc_table/doc_table.html

```diff
@@ -20,6 +20,12 @@
           class="discover-table-row"></tr>
       </tbody>
     </table>
+    <div ng-if="hits.length === sampleSize && page.last" class="discover-table-footer">
```

Comment:
In discover, [`$scope.hits`](http://github.com/elastic/kibana/blob/13ebe995cd2b8624fe86ede612148b4579b4520a/src/core_plugins/kibana/public/discover/controllers/discover.js#L422) is the total number of results. Maybe we should pass that to the doc_table and it should use that to determine if this should be shown (which means it could also tell the user how many rows are hidden).